kvdb/etcd/db.go

```diff
@@ -138,7 +139,7 @@ func NewEtcdClient(ctx context.Context, cfg Config) (*clientv3.Client,
 	context.Context, func(), error) {
 
 	clientCfg := clientv3.Config{
-		Endpoints:          []string{cfg.Host},
+		Endpoints:          strings.Split(cfg.Host, ","),
```

Comment:
I think we should also add something to the configuration description [here](https://github.com/lightningnetwork/lnd/blob/84fd911b476240bf6d172b54e734e99e38520c8d/kvdb/etcd/config.go#L17) and [here](https://github.com/lightningnetwork/lnd/blob/e8196c6feb0a7720c3e1f54deb46685f4651b311/sample-lnd.conf#L1377).

Perhaps something like `Etcd database host; supports multiple values separated by a comma`.